modules/virt-editing-vm-web.adoc

```diff
@@ -14,29 +14,28 @@
 ifeval::["{context}" == "virt-editing-vm-template"]
 :virt-vm-template:
 :object: virtual machine template
-:object-gui: Virtual Machine Template
+:object-gui: Virtual Machines
 endif::[]
 
 ifeval::["{context}" == "virt-edit-vms"]
 :virt-vm:
 :object: virtual machine
-:object-gui: Virtual Machine
+:object-gui: Virtual Machines
 endif::[]
 
 [id="virt-editing-vm-web_{context}"]
 
 = Editing a {object} in the web console
 
-Edit select values of a {object} in the *{object-gui} Overview* screen
-of the web console by clicking on the pencil icon next to the relevant field.
-Other values can be edited using the CLI.
+Edit select values of a {object} that displays in the *Virtual Machines Overview* screen of the web console by clicking on the pencil icon next to the relevant field. Other values can be edited using the CLI.
+
+Labels, annotations, and boot order are editable for both preconfigured Red Hat templates and your custom virtual machine templates. All other values are editable only for custom virtual machine templates that users have created using the Red Hat templates or the interactive wizard.
```

Comment:
@yaacov Boot order is enabled for editing, but changes in boot order is not saved? is it a bug?

Reply:
@gouyang @yaacov - My understanding is this question pertains to the behavior in the UI and does not impact the doc.

Reply:
Also this PR is only about VM templates. A new PR will be circulated for review and it focuses on VMs.

Reply:
> Boot order is enabled for editing, but changes in boot order is not saved? is it a bug?

works for me, AFAIU, if boot order not saved it's a bug and has no affect on docs

Reply:
AHHH, sorry, we are talking about common templates ?

common templates (templates provided by redhat) should not allow changing boot order.

Reply:
@yaacov This topic: modules/virt-editing-vm-web.adoc belongs to CNV-6749 where the docs focuses on VMs. This current PR CNV-9030 focuses only on VM templates. To avoid confusion and limit this current PR to VM templates, I propose that edits to this module should virt-editing-vm-web.adoc be performed as part of CNV-6749. Let me know if this works? I will make a note to make the correction you noted.

Reply:
@yaacov IIUC nothing should be editable on common templates. If you edit anything, it will be replaced back by the operator.

Reply:
@yaacov As I mentioned in the chat/email, the info referenced here has been removed from the module.

Reply:
yes, but the operator allow labels and annotations,
p.s.
it makes sense that admins (that can write to openshift name-space) will want to label or annotate some templates for